src/palm-react-native-ui-kit/components/NftDetails.tsx

```diff
@@ -20,20 +24,28 @@ import { useTranslation } from 'react-i18next'
 import { RefreshControl, ScrollView, StyleSheet, View } from 'react-native'
 
 import { useAsyncEffect } from '@sendbird/uikit-utils'
+import useAuth from 'palm-react/hooks/auth/useAuth'
+import useProfile from 'palm-react/hooks/auth/useProfile'
 
 const NftDetails = ({
   nftContract,
   tokenId,
   type,
   chain,
   item,
+  isNftInfo,
```

Comment:
what does this variable mean? different from just checking if nft is mine?

Reply:
check to my nft or someone else's nft

Reply:
I checked again and it is not being used. I will delete it.